package.json

```diff
@@ -18,11 +18,14 @@
     "test": "jest"
   },
   "dependencies": {
+    "aws-sdk": "^2.922.0",
     "axios": "^0.21.1",
     "body-parser": "^1.19.0",
     "browser-sync": "^2.26.14",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "grunt-aws-s3": "^2.0.2",
+    "grunt-s3": "^0.2.0-alpha.3",
```

Comment:
interested to see how these work

Reply:
lol so am I, haven't figured it out...yet